plugins/flowdock/app/assets/javascripts/flowdock/application.js

```diff
@@ -1,112 +1,2 @@
 //= require jquery_mentions_input/jquery.elastic.source.js
```

Comment:
yes!

On Mon, Jun 13, 2016 at 7:23 PM, Zehua Liu <notifications@github.com> wrote:

> In plugins/flowdock/app/assets/javascripts/flowdock/application.js
> <https://github.com/zendesk/samson/pull/1064#discussion_r66900432>:
>
> > @@ -1,112 +1,2 @@
> >  //= require jquery_mentions_input/jquery.elastic.source.js
>
> elastic does not seem to work on Chrome and Safari. Its last update was 5
> years ago. http://unwrongest.com/projects/elastic/
>
> We could remove it for now and figure out a different way to make the
> buddy approval message textarea looks nicer. I feel that setting a longer
> initial height would solve the most immediate UI issue.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1064/files/e876c7d763293036fd0708d1e6738939443ad7b4#r66900432>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ2JZhFb6MJYaEcQSJOJqPBNt_Oa9ks5qLhCogaJpZM4Iz2PM>
> .
>